src/components/SideBar.js

```diff
@@ -22,7 +22,7 @@ export default function SideBar(props) {
   const path = useLocation().pathname;
   useEffect(() => {
     setIsOnStudentSide(!path.includes("teacher"));
-    api.hasBookmarksInPipelineToReview((hasBookmarks) => {
+    api.hasBookmarksToReview((hasBookmarks) => {
```

Comment:
this sounds much nicer than the previous method call! this already is a strong reason for the new algorithm :)